infra/build/developer-tools/build/scripts/task_helper_functions.sh

```diff
@@ -152,11 +164,22 @@ function check_terraform() {
   echo "Running terraform validate"
   # Change to a temporary directory to avoid re-initializing terraform init
   # over and over in the root of the repository.
-  find_files . -name "*.tf" -print \
+
+  # If enable parallel, run validate in parallel
```

Comment:
@bharathkkb Is there a reason we wouldn't switch to parallel validation by default? (Fine to do this in a subsequent PR once we've tested it a bit.)